### DISCUSSION TITLE
Properties Support and Release Estimate

### DISCUSSION THREAD

**Discussion by KiljanK** [javalent/calendarium#83] [Q&A] [answered]
Hello there, i'm gonna download and try the beta when i find the time, but i had some questions beforehand.

I was wondering whether or not this plugin works with obsidian's 1.4 new way of defining frontmatter properties, as i've heard that that's broken in the fantasy-calendar plugin.

Also, is there any guesstimate on when this will be ready for release and if there will be instructions on how to use it? :)

**Comment by KiljanK** [comment]
So i guess the `v1.0.0` release means it's out of beta now!

Congrats and tons of thanks to the maintainers and contributors, you do amazing work!!!

**Comment by valentine195** [comment, marked as answer]
Sorry, didn't see this :)

Thank you for the congratulations.

> I was wondering whether or not this plugin works with obsidian's 1.4 new way of defining frontmatter properties, as i've heard that that's broken in the fantasy-calendar plugin.

This shouldn't be the case. Once Obsidian releases the properties widget API, then I am going to integrate a date-picker, but things should just work.

**Reply by KiljanK** [reply]
That sounds awesome! Hopefully, Obsidian releases the API soon.

I actually had some time to play around with the plugin over the weekend and managed to make it work with my frontmatter, i really love what you've all done here :)

However, i couldn't quite manage to set up the date format to use dots instead of dashes. Is this intended/known, or should i open up an issue? :)
